gtests/net/mptcp/mp_join/mp_join_server_v4.pkt

```diff
@@ -17,22 +17,25 @@
 +0    accept(3, ..., ...) = 4
 
 // add_addr
-+0.0    >                                .  1:1(0)  ack 1             <nop, nop,         TS val 4074418293 ecr 4074418292, add_address addr[saddr1] hmac=auto, dss dack4=1 dll=0 nocs>
-// TODO: send echo bit
-// +0.0 <                                .  1:1(0)  ack 1             <nop, nop,         TS val 4074418293 ecr 4074418292, add_address addr[saddr1],           dss dack8=1 dll=0 nocs>
++0.0    >                                .  1:1(0)  ack 1             <nop, nop,         TS val 4074418293 ecr 4074418292, add_address address_id=1 addr[saddr1] hmac=auto, dss dack4=1 dll=0 nocs>
++0.0    <                                .  1:1(0)  ack 1  win 256                                                        <add_address address_id=1 addr[saddr1] addr_echo, dss dack4=1 dll=0 nocs>
 
-
-+0.2    <                               P.  1:3(2)  ack 1  win 256    <nop, nop,         TS val 4074418292 ecr 4074410674,                mpcapable v1 flags[flag_h] key[skey, ckey] mpcdatalen 2, nop, nop>
++0.0    <                               P.  1:3(2)  ack 1  win 256    <nop, nop,         TS val 4074418292 ecr 4074410674,                mpcapable v1 flags[flag_h] key[skey, ckey] mpcdatalen 2, nop, nop>
 // MP_CAPABLE carrying data are acked with 64-bit, safer not knowing if the
 // sender will use a DSN on 64-bit or 32-bit. Later we can use the lower 32 bits
 +0.0    >                                .  1:1(0)  ack 3             <nop, nop,         TS val 4074418293 ecr 4074418292,                dss dack8=3 dll=0 nocs>
 
+
 // create subflow
-+0.0    <  addr[caddr1] > addr[saddr1]  S   0:0(0)         win 65535  <mss 1460, sackOK, TS val 448955294  ecr 0,          nop, wscale 8, mp_join_syn     backup=1 address_id=2 token=sha256_32(skey)>
-+0.0    >                               S.  0:0(0)  ack 1             <mss 1460, sackOK, TS val 448955294  ecr 448955294,  nop, wscale 8, mp_join_syn_ack backup=1 address_id=0 sender_hmac=auto>
++0.0    <  addr[caddr1] > addr[saddr1]  S   0:0(0)         win 65535  <mss 1460, sackOK, TS val 448955294  ecr 0,          nop, wscale 8, mp_join_syn     backup=1 address_id=1 token=sha256_32(skey)>
++0.0    >                               S.  0:0(0)  ack 1             <mss 1460, sackOK, TS val 448955294  ecr 448955294,  nop, wscale 8, mp_join_syn_ack backup=1 address_id=1 sender_hmac=auto>
 +0.0    <                                .  1:1(0)  ack 1  win 256    <nop, nop,         TS val 448955294  ecr 448955294,                 mp_join_ack                           sender_hmac=auto>
-+0.0    >                                .  1:1(0)  ack 1             <nop, nop,         TS val 448955294  ecr 448955294,                 dss dack8=3 nocs>
+
++0.0    >                                .  1:1(0)  ack 1             <nop, nop,         TS val 448955294  ecr 448955294,                 dss dack4=3 nocs>
+
+
++0.0  read(4, ..., 2) = 2
```

Comment:
why do you now need this `read`?

Reply:
also this is a leftover , meant to see if the MPC+data at lines 24-25 was really making 2 bytes readable by the kernel under test. I will remove it